msgpack-jackson/src/main/java/org/msgpack/jackson/dataformat/JsonArrayFormat.java

```diff
@@ -0,0 +1,33 @@
+package org.msgpack.jackson.dataformat;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.databind.introspect.Annotated;
+import com.fasterxml.jackson.databind.introspect.JacksonAnnotationIntrospector;
+
+import static com.fasterxml.jackson.annotation.JsonFormat.Shape.ARRAY;
+
+/**
+ * Provides the ability of serializing POJOs without their schema.
+ * Similar to @JsonFormat annotation with JsonFormat.Shape.ARRAY, but in a programmatic
+ * way.
+ *
+ * This also provides same behavior as msgpack-java 0.6.x serialization api.
+ *
+ * @author marenzo
```

Comment:
Please remove this `@author` tag. msgpack-java should be owned by public community :)